src/components/Workbench.tsx

```diff
@@ -281,7 +265,7 @@ const LoadExperience = () => {
                 <DeleteIcon
                   style={{ fontSize: "36px" }}
                   onClick={() => {
-                    handleDeleteDialogOpen(index, expName);       
+                    handleDeleteDialogOpen(index, exp[0]);       
                   }}
                 />
                 <MoreHorizIcon style={{ fontSize: "36px" }} />
```

Comment:
The extra buttons can be commented out or removed for now

Reply:
Will do, however the delete button is fully functional and will remain.